content/blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Kestra Raises $3 Million to Simplify and Unify Orchestration for All Engineers"
+description: "Kestra raises $3 million in funding to continue building a language-agnostic, universal open-source orchestration platform, empowering software and data engineers across industries."
+date: 2023-10-05T10:00:00
+category: News & Products Updates
+author:
+  name: Emmanuel Darras
+  image: "edarras"
+image: /blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator.jpg
+---
+
+We are thrilled to announce that Kestra has raised **$3 million of funding** to make orchestration more accessible, collaborative, and scalable. 
+
+We aim to empower a wide range of users, from software engineers to data professionals, by simplifying and unifying their automation and orchestration needs. We envision Kestra as the go-to solution for organizations seeking **simple** and **versatile** orchestration.
+
+Through a seamless UI and IDE access, all workflows — from those orchestrating mission-critical applications, business processes, and data pipelines to simple automations — can be version controlled and governed using engineering best practices.
+
+## Trusted by Companies of All Sizes
+
+Since its inception, Kestra has experienced significant traction as an open-source project, thriving with a vibrant community of users and contributors. Thousands of companies, from startups to large enterprises, including **Huawei**, **Leroy Merlin**, **Acxiom**, **Tencent**, **Gorgias**, **Sophia Genetics**, and **Decathlon**, are already orchestrating their **mission-critical workloads** with Kestra.
```

Comment:
```suggestion
Since its inception, Kestra has experienced significant traction as an open-source project, thriving with a vibrant community of users and contributors. Thousands of companies, from startups to large enterprises, including **Huawei**, **Leroy Merlin**, **Acxiom**, **Tencent**, **Gorgias**, **Sophia Genetics**, and **Decathlon**, are already orchestrating their **business-critical workloads** with Kestra.
```